apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPage.tsx

```diff
@@ -46,29 +50,51 @@ export const EditPage = () => {
         }
     };
 
+    const handleSaveDraft = async () => {
+        await savePageAsDraft(token, Number(pageId))
+            .then((response) => {
+                console.log(response);
```

Comment:
Left a console.log, also you put an 'async' on the function but then you're using .then.catch, you can use await if using async